EIPS/eip-draft_title_abbrev.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to be assigned>
+title: EthPM URI Specification
+author: Nick Gheorghita (@njgheorghita), Piper Merriam (@pipermerriam), g. nicholas d'andrea (@gnidan), Benjamin Hauser (@iamdefinitelyahuman)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-04
+requires: EIP 2678
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+A custom [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) scheme to identify an [EthPM](https://docs.ethpm.com/) registry, package, release, or specific contract asset within a release.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+When interacting with the EthPM ecosystem, users and tooling can benefit from a URI scheme to identify EthPM assets. Being able to specify a package, registry, or release with a single string makes simplifies the steps required to install, publish, or distribute EthPM packages.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+`scheme://registry_address:chain_id/package_name@package_version/json_pointer`
+
+#### `scheme`
+- Required
+- Must be one of `ethpm` or `erc2678`. If future versions of the EthPM packaging standard are designed and published via the ERC process, those ERCs will also be valid schemes.
```

Comment:
Good catch. I was incorrect here, the valid schemes are `ethpm` and `erc1319` which refers to the ethpm registry standard that the registry of the resolved URI conforms to - rather than the ethpm packaging standard (2678). In this context, `erc2942` is not a valid scheme.